ansible/roles/supervisord-init/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+    - name: Install supervisor         | Install supervisor 3.4.0
+      pip:
+        name: supervisor==3.4.0
+
+    - name: Configure supervisor       | Create a defult configuration file for supervisord
+      shell: echo_supervisord_conf > /etc/supervisord.conf
+
+    - name: Configure supervisor       | Modify supervisord.conf
+      ini_file:
+        dest: /etc/supervisord.conf
+        section: include
+        option: files
+        value: supervisord.d/*.ini
+
+    - name: Install supervisor         | Modify supervisord.conf
+      ini_file:
+        dest: /etc/supervisord.conf
+        section: unix_http_server
+        option: file
+        value: /var/run/supervisor/supervisor.sock
+
+    - name: Configure supervisor       | Modify supervisord.conf
+      ini_file:
+        dest: /etc/supervisord.conf
+        section: supervisord
+        option: logfile
+        value: /var/log/supervisor/supervisord.log
```

Comment:
It should be `/srv/logs/supervisord.log`. See https://github.com/percona/pmm-server/pull/164#discussion_r319761111

I guess that should be a variable since it is a common role.

Reply:
done